definitions/tools/cnvkit_batch.cwl

```diff
@@ -8,24 +8,28 @@ requirements:
     - class: ShellCommandRequirement
     - class: DockerRequirement
       dockerPull: "etal/cnvkit:0.9.5"
-
+    - class: ResourceRequirement
+      coresMin: 1
+      ramMin: 4000
+      tmpdirMin: 10000
+    - class: InlineJavascriptRequirement
 inputs:
     tumor_bam:
         type: File
         inputBinding:
             position: 1
     normal_bam:
-        type: File
+        type: File?
         inputBinding:
             position: 2
             prefix: "--normal"
     bait_intervals:
-        type: File
+        type: File?
         inputBinding:
             position: 3
             prefix: "--targets"
     reference:
-        type: string
+        type: string?
```

Comment:
Is a reference FASTA optional?

Reply:
It is if we decide to keep only one cnvkit batch cwl- this could be split into two, one for single sample and one for paired mode. It's optional now because in single sample mode, the only required arguments are the sample bam to be analyzed and a reference cnn file